src/puppetlabs/trapperkeeper/services/webrouting/webrouting_service_core.clj

```diff
@@ -24,6 +24,9 @@
 (def ContextHandlerOptions
   (dissoc (merge jetty9-core/ContextHandlerOptions RouteOption) :server-id))
 
+(def WebsocketHandlerOptions
+  (dissoc (merge jetty9-core/CommonOptions RouteOption) :server-id))
```

Comment:
Actually...I think we should change the name of schema to `CommonOptions` and use it for the ring handler as well. There is one option, `:redirect-if-no-trailing-slash`, that applies to all handlers. That option is not currently supported for `add-ring-handler` in the webrouting service since it's currently using `RouteOption`, which only supports the `:route-id` option, whereas this schema will also allow the `:redirect-if-no-trailing-slash` option in the webrouting service.